cf/src/main/scala/com/timeout/scalacloudformation/Output.scala

```diff
@@ -0,0 +1,7 @@
+package com.timeout.scalacloudformation
+
+case class Output(logicalId: String,
+                  Value: CfExp[String],
```

Comment:
Apparently you cannot have expressions for `Value`

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html